Assets/Scripts/Power/Domain/Saboteur.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections;
+using UnityEngine;
+using Power.Base;
+using Player.Power;
+
+namespace Power.Domain
+{
+    [CreateAssetMenu(fileName = "Saboteur", menuName = "Powers/Dark/Saboteur", order = 1)]
+    public class Saboteur : DarkPower
+    {
+        [Header("Internal state - do not assign")]
+        [SerializeField]
+        private int quantityObstacleLimit = 3;
+        private int quantityOfObstacle;
+        
+        [Header("Scaling values - tweak for balancing")]
+        [SerializeField]
+        private int[] quantityObstaclePerLevel = { 1, 2, 3 };
+        private Saboteur(int level)
+        {
+            quantityOfObstacle = quantityObstaclePerLevel[level];
+        }
+
+        override public BasePower clone(int level)
+        {
+            return new Saboteur(level);
+        }
+
+        override public IEnumerator execute(PowerExecutor executor)
+        {
+            executor.createObstacle();
+            yield return new WaitForSeconds(0);
```

Comment:
Esse valor do WaitForSeconds pode ser parametrizado para permitir que esse poder tenha uma certa "animação" na hora de criar mais de um obstáculo, similar ao que acontece com o poder LineBreaker atualmente. Entretanto isso depende do que for especificado para o poder.

Invoco @nuneeez para pesar essa decisão.

Reply:
Irei implementar a criação de mais obstaculo de acordo como nivel e o WaitForSeconds vai ser usado no tempo de criação dos obstaculos